refs/whatwg.json

```diff
@@ -257,7 +257,7 @@
         "aliasOf": "LOADER"
     },
     "WHATWG-MEDIASESSION": {
-        "aliasOf": "MEDIASESSION"
+        "obsoletedBy": "MEDIASESSION"
```

Comment:
`obsoletedBy` expects an array, see [the JSON schema](https://github.com/tobie/specref/blob/master/schemas/raw-reference.json#L202-L204). You'll also need to [provide at least a title](https://github.com/tobie/specref/blob/master/schemas/raw-reference.json#L261).